### PR TITLE
Fixes #6784 - add warning about plain text.

### LINKED ISSUE #6784: suggest adding an explanation for why "-AsPlainText -Force" is insecure
suggest adding a big red scary warning to this example, advising why this method is insecure.
(password would be logged in "$env:APPDATA\Microsoft\Windows\PowerShell\PSReadLine\ConsoleHost_history.txt")

image

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e9cd2ad3-f50b-3186-3d48-e5ea37dacd92
* Version Independent ID: cf245177-58bc-6ec9-9862-a6dcec398762
* Content: [ConvertTo-SecureString (Microsoft.PowerShell.Security) - PowerShell](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/convertto-securestring?view=powershell-7)
* Content Source: [reference/7.0/Microsoft.PowerShell.Security/ConvertTo-SecureString.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/7.0/Microsoft.PowerShell.Security/ConvertTo-SecureString.md)
* Product: **powershell**
* Technology: **powershell-cmdlets**
* GitHub Login: @TravisEz13
* Microsoft Alias: **tplunk**

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Fixes [AB#1786598](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1786598) - Fixes #6784 - add warning about plain text.

## PR Context

Select the area of the Table of Contents containing the documents being changed.

**Conceptual content**
- [ ] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [ ] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Sample scripts
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Legacy SDK

**Cmdlet reference & about_ topics**
- [x] Version 7.1 preview content
- [x] Version 7.0 content
- [x] Version 6 content
- [x] Version 5.1 content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/5.1/Microsoft.PowerShell.Security/ConvertTo-SecureString.md [hunk 1]
```diff
@@ -37,90 +34,106 @@ ConvertTo-SecureString [-String] <String> [-Key <Byte[]>] [<CommonParameters>]
...
 
 ## EXAMPLES
 
 ### Example 1: Convert a secure string to an encrypted string
 
-```
+This example shows how to create a secure string from user input, convert the secure string to an
+encrypted standard string, and then convert the encrypted standard string back to a secure string.
+
+```powershell
```
Comment (DCtheGeek) [inline reference/5.1/Microsoft.PowerShell.Security/ConvertTo-SecureString.md:56]: While updating this, clean up the prompts?

File: reference/5.1/Microsoft.PowerShell.Security/ConvertTo-SecureString.md [hunk 2]
```diff
@@ -37,90 +34,106 @@ ConvertTo-SecureString [-String] <String> [-Key <Byte[]>] [<CommonParameters>]
...
-The first command uses the *AsSecureString* parameter of the Read-Host cmdlet to create a secure string.
-After you enter the command, any characters that you type are converted into a secure string and then saved in the $Secure variable.
+The first command uses the **AsSecureString** parameter of the `Read-Host` cmdlet to create a secure
+string. After you enter the command, any characters that you type are converted into a secure string
+and then saved in the `$Secure` variable.
 
-The second command displays the contents of the $Secure variable.
-Because the $Secure variable contains a secure string, Windows PowerShell displays only the System.Security.SecureString type.
+The second command displays the contents of the `$Secure` variable. Because the `$Secure` variable
+contains a secure string, PowerShell displays only the System.Security.SecureString type.
```
Comment (DCtheGeek) [inline reference/5.1/Microsoft.PowerShell.Security/ConvertTo-SecureString.md:76]: Should the type be bold?

File: reference/5.1/Microsoft.PowerShell.Security/ConvertTo-SecureString.md [hunk 3]
```diff
@@ -37,90 +34,106 @@ ConvertTo-SecureString [-String] <String> [-Key <Byte[]>] [<CommonParameters>]
...
-It is used with ConvertFrom-SecureString and Read-Host.
-The secure string created by the cmdlet can be used with cmdlets or functions that require a parameter of type SecureString.
-The secure string can be converted back to an encrypted, standard string using the ConvertFrom-SecureString cmdlet.
-This enables it to be stored in a file for later use.
 
-If the standard string being converted was encrypted with **ConvertFrom-SecureString** using a specified key, that same key must be provided as the value of the *Key* or *SecureKey* parameter of the **ConvertTo-SecureString** cmdlet.
+The `ConvertTo-SecureString` cmdlet converts encrypted standard strings into secure strings. It can
+also convert plain text to secure strings. It is used with `ConvertFrom-SecureString` and
+`Read-Host`. The secure string created by the cmdlet can be used with cmdlets or functions that
+require a parameter of type SecureString. The secure string can be converted back to an encrypted,
```
Comment (DCtheGeek) [inline reference/5.1/Microsoft.PowerShell.Security/ConvertTo-SecureString.md:41]: Should SecureString be bold?

**Review by DCtheGeek — CHANGES_REQUESTED** [review]

File: reference/5.1/Microsoft.PowerShell.Security/ConvertTo-SecureString.md (hunk 1 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Security/ConvertTo-SecureString.md:56]: stet

File: reference/5.1/Microsoft.PowerShell.Security/ConvertTo-SecureString.md (hunk 2 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Security/ConvertTo-SecureString.md:76]: Fixed

File: reference/5.1/Microsoft.PowerShell.Security/ConvertTo-SecureString.md (hunk 3 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Security/ConvertTo-SecureString.md:41]: Fixed

**Review by DCtheGeek — APPROVED** [review]

**Pull request merged**